### PR TITLE
List scripts when prompting what to run

### PR DESCRIPTION
**Pull request by gillchristian**
This PR adds:

- prettier with precommit hook :gift: 
- list scripts (with content) when prompting what to run: https://asciinema.org/a/202391
- simple `readme.md`
- [`np`](https://github.com/sindresorhus/np) as release script

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (fvictorio) [inline .prettierignore]: Doesn't Prettier ignore `node_modules` by default?

File: index.js [hunk 2]
```diff
@@ -1,28 +1,50 @@
...
-  spawn(`npm`, ['run', script], { stdio: 'inherit' })
+function runScript(script, ...args) {
+  spawn(`npm`, ['run', script, ...args], { stdio: 'inherit' })
+}
+
+function choices(pairs) {
+  return pairs.map(([script, content]) => ({
+    value: script,
+    short: script,
+    name: chalk`{bold ${script}}: ${content}`,
```
Comment (fvictorio) [inline index.js:48]: You can use `chalk` as a literal tag??

File: package.json [hunk 3]
```diff
@@ -5,10 +5,28 @@
...
+    "scripts",
+    "run"
+  ],
+  "scripts": {
+    "format": "prettier --write '**/*.{md,json,js}'",
+    "format:changed": "pretty-quick",
+    "precommit": "pretty-quick --staged",
+    "release": "np"
+  },
+  "author": "Franco Victorio <victorio.franco@gmail.com>",
```
Comment (fvictorio) [inline package.json:20]: Add yourself too :smiley:

File: .prettierignore (hunk 1 above)
Comment (gillchristian) [reply .prettierignore]: Yes, it does. I just add 'cuz reasons.

Should I remove it?

File: index.js (hunk 2 above)
Comment (gillchristian) [reply index.js:48]: Yup. v2 added that

File: .prettierignore (hunk 1 above)
Comment (fvictorio) [reply .prettierignore]: Yeah

**Pull request merged**